examples/redis-operator.md

```diff
@@ -0,0 +1,109 @@
+    $ kubectl get po
+    NAME                                      READY   STATUS    RESTARTS   AGE
+    pmem-csi-controller-0                     3/3     Running   0          145m
+    pmem-csi-node-c82p5                       2/2     Running   0          144m
+    pmem-csi-node-dwws2                       2/2     Running   0          145m
+    pmem-csi-node-kj7gs                       2/2     Running   0          145m
+    redisoperator-688f6f4fcc-lv5h5            1/1     Running   0          18m
+    rfr-redisfailover-pmem-0                  1/1     Running   0          9m37s
+    rfr-redisfailover-pmem-1                  1/1     Running   0          6m28s
+    rfr-redisfailover-pmem-2                  1/1     Running   0          3m15s
+    rfs-redisfailover-pmem-7595857d4c-56mmz   1/1     Running   0          9m37s
+    rfs-redisfailover-pmem-7595857d4c-c69xg   1/1     Running   0          9m37s
+    rfs-redisfailover-pmem-7595857d4c-cqx4p   1/1     Running   0          9m37s
+    ```
+    **Note 1**: How to verify that the Redis instances are using the volumes provided by `pmem-csi-sc-ext4`? You can match the volumes claim name used by each Redis instance, and then match it to the current `pvc` which gets provisioned  by `pmem-csi-sc-ext4`, those needs to be properly bound. Finally, you can check the block devices in the worker nodes of your K8s cluster and match the ones under `pmem` block device, i.e.:
+    ```sh
+    $ kubectl get pvc
+    NAME                                               STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+    redisfailover-pmem-data-rfr-redisfailover-pmem-0   Bound    pvc-82ee55fc-9458-11e9-bec2-0242ac110004   100Mi      RWO            pmem-csi-sc-ext4   19m
+    redisfailover-pmem-data-rfr-redisfailover-pmem-1   Bound    pvc-f3a04c91-9458-11e9-bec2-0242ac110004   100Mi      RWO            pmem-csi-sc-ext4   16m
+    redisfailover-pmem-data-rfr-redisfailover-pmem-2   Bound    pvc-668ec3c6-9459-11e9-bec2-0242ac110004   100Mi      RWO            pmem-csi-sc-ext4   13m
+
+    $ kubectl exec rfr-redisfailover-pmem-0 -- df /data
+    Filesystem                                                   1K-blocks  Used Available Use% Mounted on
+    /dev/ndbus0region0fsdax/8999c281-9458-11e9-8866-a611eaeb0cd9     95088    72     87848   1% /data
+
+    $ kubectl exec rfr-redisfailover-pmem-0 -- mount | grep /data
+    /dev/ndbus0region0fsdax/8999c281-9458-11e9-8866-a611eaeb0cd9 on /data type ext4 (rw,relatime,dax,stripe=256)
+    ```
+
+    **Note 2**: These steps will deploy a Redis cluster with three Redis and three sentinel instances as it is shown when executing `kubectl get po`. The provided persistent volumes will have the specifications written at `persistentVolumeClaim` section of `example/redisfailover/pmem.yaml`, i.e: 100Mi of storage. Feel free to play around with the numbers of instances to be deployed at `example/redisfailover/pmem.yaml`.
```

Comment:
1) Suggest moving this paragraph to line 73 (above the new heading 3 "Verify...").
2) Use this text:  
**Note**: These steps deploy a Redis cluster with three Redis and three sentinel instances when executing the `kubectl get po` command. The provided persistent volumes use the specifications in the `persistentVolumeClaim` section of `example/redisfailover/pmem.yaml`, that is, 100Mi of storage. Feel free to play around with the numbers of instances to be deployed using the `example/redisfailover/pmem.yaml` file.

Reply:
* The **Note** has been moved as suggested.
* `kubectl get po` shows that 6 instances where created so I keep the as _it is shown when executing `kubectl get po` command_ sentence.
* There are many entries in the `persistentVolumeClaim` section, and amount of storage is just an example of it, so I keep the original sentence.